zone/client.cpp

```diff
@@ -12210,3 +12202,29 @@ void Client::PlayerTradeEventLog(Trade *t, Trade *t2)
 	RecordPlayerEventLogWithClient(trader, PlayerEvent::TRADE, e);
 	RecordPlayerEventLogWithClient(trader2, PlayerEvent::TRADE, e);
 }
+
+bool Client::GetDataBuckets()
```

Comment:
This should be mob generic

Reply:
I ended up moving this to data_buckets.cpp as a static member of DataBuckets, and have it accepting a Mob* now. Think that would be the best place for it? or should it just be a member of Mob.